src/Database/Model.php

```diff
@@ -1009,6 +1009,8 @@ public function getAttributeValue($key)
          * @event model.getAttribute
          * Called after the model attribute is retrieved
          *
+         * Note: also triggered in October\Rain\Halcyon\Model
```

Comment:
Should add this to all events here that are fired in the Halcyon model

Reply:
Yes, but I wanted to know first if this is how you want it to appear.

Reply:
Gotcha. LGTM!